app/models/place.rb

```diff
@@ -1,2 +1,8 @@
 class Place < ActiveRecord::Base
+  belongs_to :owner, class_name: 'User', foreign_key: :owner_id
+  validates :owner,  presence: true
+
+  has_attached_file :photo, :styles => { :medium => "300x300>", :thumb => "100x100>" }, :default_url => "/images/:style/missing.png"
```

Comment:
Remember to add the default images in case that the user doesn't upload one (as you have configured now it should be in the path `/images/:style/missing.png`).